### PR TITLE
Add ability to regroup svg hierarchy

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: cmd/main.go [hunk 1]
```diff
@@ -73,6 +73,7 @@ var (
 	startupMemoryProfile = flag.String("startup_memprof", "", "File path to write the memory profile of mixer startup to")
 	// Serve live profiles of the process (CPU, memory, etc.) over HTTP on this port
 	httpProfilePort = flag.Int("httpprof_port", 0, "Port to serve HTTP profiles from")
+	pinnedSvg       = flag.String("pinned_svg", "", "When exist, this will be a top level svg and other other top level svg will be grouped together")
```
Comment (pradh) [inline cmd/main.go:76]: nit: "When set, this will be the ..."
nit: typo "other other"

File: internal/server/statvar/statvar_group.go [hunk 2]
```diff
@@ -33,6 +33,7 @@ const (
 	// SvgRoot is the root stat var group of the hierarchy. It's a virtual entity
 	// that links to the top level category stat var groups.
 	SvgRoot         = "dc/g/Root"
+	SvgShadowRoot   = "dc/g/ShadowRoot"
```
Comment (pradh) [inline internal/server/statvar/statvar_group.go]: Describe what ShadowRoot is for?

**Review by pradh — APPROVED** [review]

File: cmd/main.go (hunk 1 above)
Comment (shifucun) [reply cmd/main.go]: Done

**Review by shifucun — COMMENTED** [review]
Thanks!

File: internal/server/statvar/statvar_group.go (hunk 2 above)
Comment (shifucun) [reply internal/server/statvar/statvar_group.go]: Added more comments

**Pull request merged**